File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.cs
using System;

public class Solution {
    public int solution(string t, string p) {
        int answer = 0;
        int len = t.Length - p.Length + 1;
        long pNum = long.Parse(p);
        for(int i = 0; i < len; i++)
        {
            string temp = t.Substring(i, p.Length);
            if(long.Parse(temp) <= pNum)
            {
                answer++;
            }
        }
        return answer;
    }
}

File: 프로그래머스/1/147355. 크기가 작은 부분문자열/크기가 작은 부분문자열.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string t, string p) {
    int answer = 0;
    int len = p.size();
    
    for(int i = 0; i <= t.size() - len; i++)
    {
        if(stoll(t.substr(i, len)) <= stoll(p))
            answer++;
    }
    
    return answer;
}